app/components/login/LoginForm.js

```diff
@@ -0,0 +1,78 @@
+import React, { PropTypes } from 'react';
+import Formsy from 'formsy-react';
+import { Input, Checkbox } from 'formsy-react-components';
+import autoBind from '../../lib/autoBind';
+
+class LoginForm extends React.Component {
+  constructor(props, context) {
+    super(props, context);
```

Comment:
Why are you using `context` here?

Reply:
I thought the same... But I copy paste the code from UserForm and it was there as well... I think it can be removed but I am not sure... I should give it a try.